packages/web-react/src/components/Radio/figma/Radio.figma.tsx

```diff
@@ -0,0 +1,28 @@
+import figma from '@figma/code-connect';
+import React from 'react';
+import Radio from '../Radio';
+
+figma.connect(Radio, '<FIGMA_FILE_ID>?node-id=830%3A1204', {
+  props: {
+    isChecked: figma.enum('Interaction State', {
+      Selected: true,
+      Unselected: false,
+    }),
```

Comment:
Why is this status different from Checkbox?

https://github.com/lmc-eu/spirit-design-system/pull/2322/changes#diff-d4549b34854fc589e7a5f8b252b124a9e601af09a791850f8aee213ab4e1b07bR8